cni_stop.sh

```diff
@@ -113,8 +113,26 @@ function unload_module {
 
 let status=0
 
+function load_inbox_drivers {
```

Comment:
as you are just loading mlx5_core and ib_core, perhaps rename it to `load_core_drivers`